addons/sale/models/sale_order.py

```diff
@@ -326,7 +326,10 @@ def _compute_note(self):
             order = order.with_company(order.company_id)
             if order.terms_type == 'html' and self.env.company.invoice_terms_html:
                 baseurl = html_keep_url(order._get_note_url() + '/terms')
+                context = {'lang': order.partner_id.lang or order.env.user.lang}
                 order.note = _('Terms & Conditions: %s', baseurl)
+                del context
+                return order.note
```

Comment:
you should remove the return, in 16.0:

- this method doesn't return a value (it doesn't cause an issue but it's not useful so we should not do it)
- this method is called with several records, return would skill all the other records which would cause an error (because in a compute method, the field needs to be computed for all the records that are given to it)

Reply:
@nle-odoo 
I just updated it, can I make it ready for review?